src/applications/financial-status-report/sass/financial-status-report.scss

```diff
@@ -19,21 +19,6 @@
   margin: 0;
 }
 
-.debt-card {
```

Comment:
this class does not seem to be in use and it is used in conjunction with a Font Awesome icon so I deleted it